### DISCUSSION TITLE
Dropdown menu under `Star` in repositories isn't centered on Firefox

### DISCUSSION THREAD

**Discussion by hwittenborn** [community/community#10195] [Other Feature Feedback, Questions, & Ideas]
When using Firefox, the dropdown icon for the `Star` button isn't centered like `Watch` is:

![image](https://user-images.githubusercontent.com/74838472/149670151-3c17340b-9f56-4cdc-9ade-e950074e1d57.png)

**System Info:**
OS: Ubuntu 20.04
Browser: Firefox 96.0

**Comment by Dosunmu12345** [comment]
#10216 #3969

**Reply by Dosunmu12345** [reply]
#10123 #3969 #5596

**Reply by Dosunmu12345** [reply]
#10216 #5596

**Comment by Dosunmu12345** [comment]
#3969 #10123

**Comment by History-exe** [comment]
Same thing happens. That's annoying.

Ubuntu 20.04.3
Firefox 100.0
